dev/run-test-container.sh

```diff
@@ -0,0 +1,4 @@
+ROOT_DIR=$(git rev-parse --show-toplevel)
+docker build -f "${ROOT_DIR}/dev/Dockerfile.test" -t mlflow-test-env "$ROOT_DIR"
+docker run -v "${ROOT_DIR}"/tests:/app/tests -v "${ROOT_DIR}"/mlflow:/app/mlflow --name "mlflow-test" -it mlflow-test-env
```

Comment:
```suggestion
docker run -v "${ROOT_DIR}":/app --name "mlflow-test" -it mlflow-test-env
```

Can we mount the entire repository? The repository root contains several configuration files such as `pyproject.toml`.

Reply:
🤔 ideally we should avoid mounting everything - e.g. somebody would want to develop UI locally and mounting `node_modules` in this scenario could degrade performance. @harupy Is there a finite list of files that are vital here?

Reply:
Got it. Let's just mount the following files that are required for python:

- `pyproject.toml`
- `pylintrc`
- `pytest.ini`
- `conftest.py`